src/main/kotlin/plugin/task/ParsePluginYaml.kt

```
@@ -5,65 +5,94 @@
-    private fun readFromFile(): PluginYaml {
-        if (!pluginYamlFile.isPresent) return PluginYaml()
-
-        val file = pluginYamlFile.get().asFile
-        val pluginYaml = try {
-            file.inputStream().use { yaml.decodeFromStream<PluginYaml>(it) }
+    private fun readFromFile(): BukkitPluginYamlDefaults? {
+        return try {
+            pluginYamlFile.get().asFile
+                .inputStream()
+                .use { yaml.get().decodeFromStream<BukkitPluginYamlDefaults>(it) }
         } catch (cause: EmptyYamlDocumentException) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)